src/util/AllocatorWithLimit.h

```diff
@@ -1,15 +1,18 @@
-//
-// Created by johannes on 27.04.20.
-//
+// Copyright 2020, University of Freiburg,
+// Chair of Algorithms and Data Structures.
+// Author: Johannes Kalmbach (April of 2020,
+// kalmbach@informatik.uni-freiburg.de)
 
-#ifndef QLEVER_ALLOCATORWITHLIMIT_H
-#define QLEVER_ALLOCATORWITHLIMIT_H
+#pragma once
+
+#include <absl/strings/str_cat.h>
 
 #include <atomic>
 #include <functional>
 #include <memory>
 
 #include "Synchronized.h"
```

Comment:
Also unify this include .

Reply:
That would go against our code style.

Reply:
I meant the `"util/Synchronized.h"`.

Reply:
Ah, yeah, that makes more sense.

Reply:
Done, see [44b5b81](https://github.com/ad-freiburg/qlever/pull/1112/commits/44b5b81787905bfc1f78f2dd7395fa0ab33bdafa).